rollup.config.js

```diff
@@ -1,18 +1,40 @@
 import babel from 'rollup-plugin-babel'
 import commonjs from 'rollup-plugin-commonjs'
+import resolve from '@rollup/plugin-node-resolve'
+import {terser} from 'rollup-plugin-terser'
+import visualizer from 'rollup-plugin-visualizer'
+
+// NOTE: this can be removed once the next version of rollup-plugin-commonjs is released
+const namedExports = {
+  'prop-types': ['object', 'func', 'oneOfType', 'node', 'bool', 'string', 'any', 'arrayOf'],
```

Comment:
Does this need to be updated every time we add a new propType?

Reply:
No, this is because the reach libraries do weird things with their imports. `rollup-plugin-commonjs` just implemented code that will do this automatically but they haven't made a release yet.